open_spiel/games/phantom_go/phantom_go_board.cc

```diff
@@ -481,9 +419,45 @@ void GoBoard::SetStone(VirtualPoint p, GoColor c) {
   board_[p].color = c;
 }
 
+std::array<GoColor, kMaxBoardSize* kMaxBoardSize> PhantomGoBoard::getObservationByID(int player_id) const
```

Comment:
Use capital letters for method names: https://google.github.io/styleguide/cppguide.html#Function_Names